DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/ContractDetails/Index.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@inject IConfiguration Configuration
+
+@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.ContractDetails
+@using Microsoft.Extensions.Configuration
+@model ContractDetailsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/contractDetails.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Contract details";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+    <div class="nhsuk-width-container">
+      <ol class="nhsuk-breadcrumb__list">
+        <li class="nhsuk-breadcrumb__item">
+          <a class="nhsuk-breadcrumb__link" asp-controller="Dashboard" asp-action="Index">Centre dashboard</a>
+        </li>
+      </ol>
+      <p class="nhsuk-breadcrumb__back">
+        <a class="nhsuk-breadcrumb__backlink" asp-controller="Dashboard" asp-action="Index">Back to centre dashboard</a>
+      </p>
+    </div>
+  </nav>

```

Comment:
Same comment as on centre ranking about commonising this into a partial - feel free to create partial in the ticket that you merge first that then reuse in the second one :)
Might also be worth giving @DanBloxham-sw a heads up as he will do Top courses :)